### ISSUE TITLE
音频转文本时，转为的文本出现上下文一模一样的情况，怎么办呢？

### ISSUE THREAD

**Issue by xxm1668** [SYSTRAN/faster-whisper#595]

**Comment by trungkienbkhn** [comment]
Which model are you using? You should use large models (v2, v3) for best quality.
Besides, have you tested with other languages, for example English?

**Comment by xxm1668** [comment]
can you share your code?

**Comment by trungkienbkhn** [comment]
I use code from this [project](https://github.com/MahmoudAshraf97/whisper-diarization) and do not change any thing.
Below is my command to run:
`python3 diarize.py -a chinese.m4a --whisper-model tiny --device cpu`

**Comment by blackpolarz** [comment]
I don't think this is a problem in faster-whisper but rather whisper large v3 model or the speech diarization algorithm you are using. Might want to check with whoever's speech diarization model you are using. There were a lot of complains that Whisper large v3 is not suitable for chinese transcription. You might want to use whisper large v2 instead.
这个问题应该跟faster-whisper 无关。主要还是whisper large v3 和 你用的speech diarization。你应该去问问那个弄speech diarization 的负责人。whisper large v3 据说不是很适合中文转文本，如果要用还是用whisper large v2 比较合适。

**Comment by xxm1668** [comment]
收到，我试试哈

**Comment by syumi-1** [comment]
condition_on_previous_text: If True, the previous output of the model is provided as a prompt for the next window; disabling may make the text inconsistent across windows, but the model becomes less prone to getting stuck in a failure loop, such as repetition looping or timestamps going out of sync.

try to set it

` segments, info = model.transcribe(audio=f'whispertemp.mp3',
                                      beam_size=5,
                                      language=language,
                                      condition_on_previous_text=False,
                                      vad_filter=True,
                                      vad_parameters=dict(min_silence_duration_ms=500))`

**Comment by HollrayChan** [comment]
我使用large-v3转录中文音频文件的时候，到120s后就容易出现内容重复，用v2就不会出现，两者效果不仔细对比，基本没什么差别
